packages/mobile/src/services/audius-backend-instance.ts

```diff
@@ -17,8 +20,67 @@ declare global {
   }
 }
 
+const libsInitEventEmitter = new EventEmitter()
+
 export let audiusLibs: AudiusLibs
 
+const LIBS_INITTED_EVENT = 'LIBS_INITTED_EVENT'
+
+/**
+ * Wait for the `LIBS_INITTED_EVENT` or pass through if there
+ * already exists a mounted `window.audiusLibs` object.
+ */
+const waitForLibsInit = async () => {
+  // If libs is already defined, it has already loaded & initted
+  // so do nothing
+  // @ts-ignore
```

Comment:
i think you can remove the @ts-ignore artifacts, but nbd

Reply:
👍